### ISSUE TITLE
1.20.2 and 1.20.3 missing metadataKeys in entities.json.

### ISSUE THREAD

**Issue by wgaylord** [PrismarineJS/minecraft-data#817]
Seems that the minecraft-data-generator-server doesn't output that information, and the tests don't check that it exists. 

I will look into getting this information into the entities.json on both versions and look into if minecraft-data-generator-server can generate it.

**Comment by wgaylord** [comment]
Also just looked and Burger also does not extract this info, so it might be manual.

**Comment by rom1504** [comment]
You can look at git blame info to find the commit that added it previously

**Comment by rom1504** [comment]
https://github.com/PrismarineJS/minecraft-data/pull/710 info on how to extract
Would be good to automate that

**Comment by extremeheat** [comment]
Yeah, looks like it was not added to the doc. Running the extraction script in tools/js will generate it. The reason there is no test is that it's technically optional, as we don't have the data for all the versions. I guess a test could be done to check if versions newer than X have the field, but maybe the better solution would be to try and get all the data back.

**Comment by extremeheat** [comment]
mineflayer is independent to minecraft-data, so what's optional in minecraft-data may indeed not be optional for mineflayer

**Comment by rom1504** [comment]
Adding more data validation and in general data quality tests to Minecraft data would definitely help.
It can be done as a failing test but it can also be implemented as a report that is sent by a bot.

See https://github.com/PrismarineJS/prismarine-packet-dumper/pull/82#issuecomment-1872331313 as an example of what a quality report might look like
